auth.go

```diff
@@ -62,6 +67,42 @@ func (a *Auth) CheckPermission(scope, userID string) error {
 	return ErrNotAllowed
 }
 
+// GetUsers returns a list of userIDs having one or more permission scopes
+func (a *Auth) GetUsers() ([]string, error) {
+	keys, err := a.store.Keys()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to load permissions")
+	}
+
+	a.logger.Debug("Retrieving all userIDs")
+
+	var userIDs []string
+	for _, key := range keys {
+		userID, err := a.userFromKey(key)
+		if err != nil {
+			return nil, errors.Wrap(err, "failed to parse userID from key")
+		}
+		userIDs = append(userIDs, userID)
+	}
+
+	return userIDs, nil
+}
+
+// GetUserPermissions returns the permission scopes for a specific user
+func (a *Auth) GetUserPermissions(userID string) ([]string, error) {
```

Comment:
What do you think about renaming this to `UserPermissions`?

https://golang.org/doc/effective_go.html#Getters

Reply:
I've updated the function name as per your suggestion in https://github.com/go-joe/joe/pull/28/commits/2babcb8d6b43e6663aa39f58f24b9ba5317009f4